_data/team/roles/non_core_library_maintenance.yml

```
@@ -0,0 +1,84 @@
+role: Non-core library maintenance
+description: |
+  The non-core library maintenance team is responsible for
+  managing other packages that are directly
+  maintained by the MDAnalysis organization.
+
+tasks:
+  - Managing issues and pull-requests for sub-projects
+  - Managing new features and updates
+  - Managing maintenance
+  - Working with CI and CD team and infrastructure
+subroles:
+
+  - subrole: distopia
+    current_leads: [Richard Gowers,Hugo MacDermott-Opeskin]
+    current_members: [Richard Gowers,Hugo MacDermott-Opeskin, Rocco Meli]
+    historical_members:
+      - Richard Gowers
+      - Hugo MacDermott-Opeskin
+    
+
+  - subrole: cookiecutter-mdakit
+    current_leads:
+      - Lily Wang
+    current_members: []
```

```suggestion
    current_members: [Irfan Alibay]
```
Going by how tightly bound to mdakits this is, I suppose I should be here.

---

Adding!